robottelo/virtwho_utils.py

```diff
@@ -249,7 +249,7 @@ def deploy_validation():
     return hypervisor_name, guest_name
 
 
-def deploy_configure_by_command(command, debug=False, org=DEFAULT_ORG):
+def deploy_configure_by_command(command, debug=False, org='Default_Organization'):
```

Comment:
Why don't we use `DEFAULT_ORG`for this function, I saw `DEFAULT_ORG` is used by 
`def get_configure_command(config_id, org=DEFAULT_ORG)`, so a little curious :)

Reply:
Then, the result would be:
```
[root@bootp-73-131-205 ~]# subscription-manager register --org=Default Organization --environment=Library --username=admin --password=admin
cannot parse argument: Organization
[root@bootp-73-131-205 ~]# 
```
So, we should set 'Defaut_Organization' as default org, and this situation will not happen when deploying the virt-who-plugin.

Reply:
Make sense to me, Thanks